### DISCUSSION TITLE
Managing Genres

### DISCUSSION THREAD

**Discussion by PixyDon** [adultmm/AdultMediaManager#83] [General]
Anybody got an easy work around for Genre Management.

Seem to be scraping all sorts recently, but can’t get my head around a fast way to remove those unwanted, or the Genre Mapping function??

**Comment by Charlotte-br560** [comment]
One quick workaround is to use [Crawlbase](https://crawlbase.com/) for efficient genre management. It offers intuitive tools for scraping and allows for easy removal of unwanted genres. Additionally, consider exploring the Genre Mapping function within Crawlbase for streamlined genre organization.

**Comment by wkearney99** [comment]
Genre mapping in EMM isn't difficult but can be time-consuming.  The idea is you're selecting a genre on the left and mapping that against one, more than one, or no genres on the right.  The UI for it is clunky.  You have to scroll A LOT if you're re-mapping genres.  Like getting rid of a ton of 'yearly award' genres.  Select it on the left, scroll on the right to uncheck.  Or use/create another genre and point all of them to it.  Select left, scroll right, check/uncheck... rinse.. repeat...

You leave any unwanted ones you want to remove IN THE LEFT LIST, and then check no other genres in the right list.  This tells EMM that it should continue to recognize the genre if/when it sees it again, but not remap it to anything else.

It'd be helpful of the right listbox gathered currently mapped genres.  As in, click on one in the left listbox and the right box would automagically bring any mapped ones to the top.  It doesn't do this.  You have to scroll through the WHOLE list to determine if there's any other mapping going on.  

Once you've got your remapping set, click the OK button and it will then march through the entries and do the remapping, and (iirc) re-write the NFO files.
